src/aircrafts/F16/fcs.jl

```diff
@@ -65,7 +72,7 @@ function set_thtl(fcs::F16FCS, value, allow_out_of_range=false, throw_error=fals
     set_value(fcs.thtl, value)
     min, max = get_value_range(fcs.thtl)
     range = max - min
-    set_value(fcs.cpl, tgear(value), allow_out_of_range, throw_error)
+    set_value(fcs.cpl, Aircrafts.tgear(value), allow_out_of_range, throw_error)
```

Comment:
This was calling this method https://github.com/parthp08/FlightMechanics.jl/blob/master/src/aircrafts/F16/propulsion.jl#L50 I think it is ok like that because of the include order. No implicit reference is necessary.

Reply:
yes, you are right.  will remove it.